__tests__/app.test.js

```diff
@@ -117,19 +117,79 @@ describe('appTests', () => {
                     }
                 })
         })
-            test('GET: 200 /api/reviews/:review_id/comments | returns an empty array for an existing review_id with no comments', () => {
-                return request(app)
-                    .get('/api/reviews/5/comments')
-                    .expect(200)
-                    .then(({body})=>{
-                        const comments = body.reviewComments;
-                        expect(Array.isArray(comments)).toBe(true)
-                        expect(comments.length).toBe(0)
-                    })
+        test('GET: 200 /api/reviews/:review_id/comments | returns an empty array for an existing review_id with no comments', () => {
+            return request(app)
+                .get('/api/reviews/5/comments')
+                .expect(200)
+                .then(({body})=>{
+                    const comments = body.reviewComments;
+                    expect(Array.isArray(comments)).toBe(true)
+                    expect(comments.length).toBe(0)
+                })
+        })
+    })
+
+    describe('PATCH 202 /api/reviews/:review_id', () => {
+        test('increments votes for a review by a given amount ', () => {
+            const testPatch = {
+                "inc_votes": 5 
+            }
+
+            return request(app)
+                .patch('/api/reviews/2')
+                .send(testPatch)
+                .expect(202)
+                .then(({body}) => {
+                    console.log(body.updatedReview, "<<< should have returned updated review")
+                    expect(body.updatedReview.votes).toBe(10)
+
+                    expect(Object.prototype.toString.call(body.updatedReview)).toBe('[object Object]')
+
+                    expect(Object.keys(body.updatedReview).length).toBe(9)
+
+                    expect(body.updatedReview).toHaveProperty('review_id',expect.any(Number))
+                    expect(body.updatedReview).toHaveProperty('title',expect.any(String))
+                    expect(body.updatedReview).toHaveProperty('category',expect.any(String))
+                    expect(body.updatedReview).toHaveProperty('designer',expect.any(String))
+                    expect(body.updatedReview).toHaveProperty('owner',expect.any(String))
+                    expect(body.updatedReview).toHaveProperty('review_body',expect.any(String))
+                    expect(body.updatedReview).toHaveProperty('review_img_url',expect.any(String))
+                    expect(body.updatedReview).toHaveProperty('created_at',expect.any(String))
+                    expect(body.updatedReview).toHaveProperty('votes',expect.any(Number))
```

Comment:
You are already checking this on line 144, also i would expect that the review_id is 2 since thats the one were voting on